python/cuml/test/test_fil.py

```diff
@@ -18,18 +18,48 @@
 import os
 
 from cuml import ForestInference
-from cuml.test.utils import array_equal, unit_param, \
+from cuml.test.utils import get_handle, array_equal, unit_param, \
```

Comment:
i think instead of adding a new test it will be good if we just modify the test, `test_rf_classification_multi_class` present in `test_random_forest.py` to use `predict_model='GPU'`. This will help keep the number of tests limited and prevent duplicate test cases.

Reply:
BTW, if we're trying to reduce the number of tests, why don't we combine the predict with predict_proba tests? would save on model training time